python_web3_wallet/frontend/src/constants.ts

```diff
@@ -1 +1,2 @@
-export const AGENT_COMMUNICATION_CONTRACT = '0xd422e0059ed819e8d792af936da206878188e34f';
\ No newline at end of file
+export const AGENT_COMMUNICATION_CONTRACT =
+  "0xd422e0059ed819e8d792af936da206878188e34f"
```

Comment:
Yesterday I re-deployed 
```suggestion
  "0xe9dd78FF297DbaAbe5D0E45aE554a4B561935DE9"
```

For the new deployment that allows popping at any index 🙏

Reply:
Thanks. I don't see an easy way to reference this contract, if there is, happy to re-define it instead of hardcoding it.